checks.lua

```diff
@@ -6,6 +6,14 @@ local _qualifiers_cache = {
     -- },
 }
 
+local function is_tarantool()
+    local rc = false
+    if _G['_TARANTOOL'] ~= nil then
+        rc = true
+    end
+    return rc
```

Comment:
It could be simplified to `return _G['_TARANTOOL'] ~= nil`

Reply:
FIxed.